### PR TITLE
Added EventViewer logs path

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by illfated** [comment]
Ref. #3205

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: mdop/mbam-v25/client-event-logs.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ms.date: 06/16/2016
 
 # Client Event Logs
 
-
+MBAM Client event logs are located in Event Viewer – Applications and Services Logs – Microsoft – Windows – MBAM - Operational path.
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/client-event-logs.md]: Copy edit - no changes required.

**Comment by mypil** [comment]
@jamiejdt - Can you please assist to check, approve and merge the content for this PR based on issue #3205?

Thank you.

**Comment by mypil** [comment]
@jamiejdt - I would like to follow up on this.  Can you please assist to check, approve and merge the content for this PR based on issue #3205?

Thank you.

**Comment by mypil** [comment]
@jamiejdt - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3205?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR or do you happen to know who we can reach out to review/merge the PR content? Thank you.

**Pull request merged**